### PR TITLE
Feat/reverse registrar

### PR DESCRIPTION
**Pull request by tomicvladan**
A reverse registrar contract has been added and the library updated to support it.

### CODE REVIEW

File: js-library/src/contracts/FDSNameResolver/FDSNameResolver.dbg.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "_format": "hh-sol-dbg-1",
+  "buildInfo": "../../build-info/041b9b8ba8f9846aee37e132a76f9297.json"
```
Comment (IgorShadurin) [inline js-library/src/contracts/FDSNameResolver/FDSNameResolver.dbg.json:3]: what file does it refer to? It has not been added to the project

File: js-library/src/contracts/FDSReverseRegistrar/FDSReverseRegistrar.dbg.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "_format": "hh-sol-dbg-1",
+  "buildInfo": "../../build-info/27abc60de1d80024e2d6c981621ac5b2.json"
```
Comment (IgorShadurin) [inline js-library/src/contracts/FDSReverseRegistrar/FDSReverseRegistrar.dbg.json:3]: what file does it refer to? It has not been added to the project

**Review by IgorShadurin — APPROVED** [review]

File: js-library/src/contracts/FDSNameResolver/FDSNameResolver.dbg.json (hunk 1 above)
Comment (tomicvladan) [reply js-library/src/contracts/FDSNameResolver/FDSNameResolver.dbg.json:3]: These files are generated by hardhat when contracts are compiled. Maybe we don't need the dbg.json files, but the other json contain abi section that is required.

**Comment by IgorShadurin** [comment]
@tomicvladan Is there something that needs to be fixed here, or can we merge?

**Comment by tomicvladan** [comment]
Hey @IgorShadurin , I think it can be merged, although we still need to deploy the contracts to testnets and test them there.

**Pull request merged**